modules/kubernetes/pages/quickstart-with-aks.adoc

```diff
@@ -10,23 +11,35 @@ This is a quickstart guide for deploying TigerGraph single servers and clusters
 * Provision Kubernetes cluster on AKS with nodes that meet the xref:installation:hw-and-sw-requirements.adoc[hardware and software requirements] to run TigerGraph.
 * https://kubernetes.io/docs/tasks/tools/[Install `kubectl` on your machine,] and make sure your local `kubectl` version is within one minor version's difference from the `kubectl` version on your cluster.
 * https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#connect-to-the-cluster[Configure `kubectl` for ASK cluster access].
+* Ensure your have the following permissions in your Kubernetes context:
+** Create and delete Pods, Services, StatefulSets, and ConfigMaps
+** Create and delete Jobs, CronJobs
+** Create and delete Service Accounts, roles and role bindings
 
+
+[#_single_server_deployment]
 == Single-server deployment
 
 This section describes the steps to deploy, verify, and remove a single-server deployment of TigerGraph on AKS.
 
 === Deploy single server
 
-*Step 1: Generate deployment manifest*. Clone the https://github.com/tigergraph/ecosys.git[TigerGraph ecosystems repository] and change into the `k8s` directory. You can edit the `kustimization.yaml` file in the `aks` folder to change the namespace and image name for your deployment. The default namespace is `default`, and the default image is the official docker image for TigerGraph 3.2.0. No need to edit the files if no changes are needed.
+*Step 1: Generate deployment manifest*.
+Clone the https://github.com/tigergraph/ecosys.git[TigerGraph ecosystems repository] and change into the `k8s` directory.
+You can edit the `kustimization.yaml` file in the `aks` folder to change the namespace and image name for your deployment.
+The default namespace is `default`, and the default image is the official docker image for TigerGraph in the latest.
+No need to edit the files if no changes are needed.
 
-Next, run the `./tg` script in the `k8s` directory to generate the deployment manifest for a single-server deployment. A `deploy` directory will be created automatically and you should find the manifest named `tigergraph-aks.yaml` in the directory.
+Next, run the `./tg` script in the `k8s` directory to generate the deployment manifest for a single-server deployment.
+A `deploy` directory will be created automatically, and you should find the manifest named `tigergraph-aks.yaml` in the directory.
```

Comment:
Change to `tigergraph-aks-[deployment].yaml` (same goes for eks and gke)

Reply:
I added a note that the actual filename will contain the namespaces provided by the user.